pouchdb-core/pouchdb-core.d.ts

```diff
@@ -1,8 +1,11 @@
-// Type definitions for pouchdb-core v5.4.4
+// Type definitions for pouchdb-core v6.0.7
 // Project: https://pouchdb.com/
-// Definitions by: Andy Brown <https://github.com/AGBrown>, Brian Geppert <https://github.com/geppy>, Frederico Galvão <https://github.com/fredgalvao>
+// Definitions by: Jakub Navratil <https://github.com/trubit>, Andy Brown <https://github.com/AGBrown>, Brian Geppert <https://github.com/geppy>, Frederico Galvão <https://github.com/fredgalvao>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+/// <reference path="../es6-promise/es6-promise.d.ts" />
+/// <reference path="../node/node.d.ts" />
```

Comment:
NodeJS.EventEmitter, search code. Api mentions nodejs event emitter. Its possible to declare it same as nodejs maybe? But thats double work possibly